src/App.css

```diff
@@ -1,5 +1,102 @@
-* {
-  box-sizing: border-box;
-  margin: 0;
-  padding: 0;
+body {
+  background-color: #f5f6fa;
+}
+
+.book_container {
+  width: 100%;
+  list-style-type: none;
+  margin-left: 0%;
+  padding-left: 0%;
+}
+
+/* card body */
+.card {
+  width: 100%;
+  margin-top: 5%;
+  margin-bottom: 5%;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  transition: 0.4s;
+}
```

Comment:
- [ ] You have done well with the design 👍. However, your design is quite different from Zeplin and it is required that all elements in the design should be added, and also features that haven’t been implemented yet should be added as a placeholder and do not need to be implemented. Please let's make our design match that of Zeplin 😄. I have highlighted the things that you need to work on to be able to match Zeplin's design below.

Your Implementation | Zeplin Design
:---:|:---:
![](https://user-images.githubusercontent.com/81468675/150560689-8e944340-e4d6-47b8-8474-1eff8cf318dd.png) | ![](https://user-images.githubusercontent.com/81468675/150560945-9f835192-1b19-49f4-b37e-eb592dc7a78c.jpg)

Reply:
Thank you for that. I have implemented the required changes.